docs/spl-docs/eval-command.md

```diff
@@ -0,0 +1,113 @@
+# eval command
+
+## Description
+
+The `eval` command computes an expression and assigns the outcome to a field within the search results. If the specified field name does not correspond to an existing field in the output, the command introduces a new field to the search results. Conversely, if the specified field name already exists, the command overwrites that existing field with the values derived from the `eval` expression. The `eval` command is capable of processing mathematical, string, and boolean expressions.
+
+For executing multiple `eval` expressions within a single search, you can separate them with commas. The search handles these expressions sequentially from left to right, allowing the use of fields evaluated earlier in expressions that follow.
+
+## Syntax
+
+The required syntax is in **bold**.
+
+**eval**\
+**\<assignment_expression\>** ["," \<assignment_expression\> ]...
+
+### Required Arguments
+
+#### \<assignment_expression\>
+
+**Syntax:** `<field>=<expression>`\
+**Description:** \
+`<field>` refers to the name of the destination field that will contain the result of the evaluation of `<expression>`.\
+If the field name already exists in your record, it will be overwritten with the evaluated result.\
+Otherwise, a new field will be created. Avoid using reserved keywords as the field name.
+
+`<expression>` is a `<string>` that may include a mix of values, variables, operators, and functions. These components are evaluated together to compute the final result for the target `<field>`.
+- Expressions cannot concatenate string values or fields using the `+` operator directly.
+- Expressions such as `... | eval new_address = address + " USA"` are not supported.
```

Comment:
> Expressions cannot concatenate string values or fields using the `+` operator directly.

I think you can use `.` to concatenate strings. If that's correct, it would be good to mention that

Reply:
addressed in the latest commit